CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- `MessagingSession` reconnect loop did not break on error past reconnect deadline. Infinite reconnect loop was caused due to firstConnectionAttemptTimestamp not being set as startedConnectionAttempt was not invoked. Check https://github.com/aws/amazon-chime-sdk-js/issues/2372 for details.
```

Comment:
nit: Back tick `firstConnectionAttemptTimestamp` and `startedConnectionAttempt`.